src/main/pages/che-7/installation-guide/assembly_airgap-installation.adoc

```diff
@@ -0,0 +1,161 @@
+
+However, the offline plugin registry (see link:{site-baseurl}che-7/airgap-install/#offline-plugin-registry[above]) includes a list of all images referenced in its plugins. The easiest way to get a list of images required for plugins is by reading this file -- assuming an offline build of the plugin registry is tagged `<my-offline-registry>`, the following command will print a list of images referenced in this image:
+
+[source,bash]
+----
+docker run -it --rm --entrypoint cat <my-offline-registry> /var/www/html/v3/external_images.txt
+----
+
+
+#### Devfile base images
+
+In addition to the sidecar containers used by plugins, every Che workspace comes with one or more 'base' images, which contain the development dependencies for the projects being built. As the sample devfiles included in the devfile registry refer to images suited for this purpose, it is necessary to pull these images as well to be able to use the samples.
+
+Similarly to the plugin registry, an offline devfile registry (see link:{site-baseurl}che-7/airgap-install/#offline-devfile-registry[above]) contains a list of all images referenced in its devfiles:
+
+[source,bash]
+----
+docker run -it --rm --entrypoint cat <my-offline-registry> /var/www/html/devfiles/external_images.txt
+----
+
+where `<my-offline-registry>` is the image name and tag of the custom devfile registry.
+
+
+## Configuring Che to run in airgapped mode
+
+Once all the required images have been placed in an image repository that is visible to the cluster where Che is to be deployed, the final step is to configure Che and related containers to refer to these images instead of the default.
+
+The steps below assume that the images described in link:{site-baseurl}che-7/airgap-install/#pulling-images-used-by-che-to-a-private-registry[the pulling section] have been pushed to a registry `<my-internal-registry>` under organization `<my-organization>`, with the same image names and tags. Additionally the placeholders `<my-offline-devfile-registry>` and `<my-offline-plugin-registry>` are used to refer to the offline registries built according to the steps link:{site-baseurl}che-7/airgap-install/#offline-registries[above]. Finally, placeholder `<ver>` is used in place of the release of Che that is being deployed.
+
+Currently, the only supported method of deploying Che in airgap mode is through the Che operator
+
+### Using the Che Operator
+
+The `CheCluster` custom resource that is created by the operator includes a few fields to make deploying an airgapped instance of Che easier. When creating the `CheCluster`, it's sufficient to specify the fields
+
+[source,yaml]
+----
+# [...]
+spec:
+  server:
+    devfileRegistryImage: '<my-offline-devfile-registry>'
+    pluginRegistryImage: '<my-offline-plugin-registry>'
```

Comment:
It isn't necessary to set `devfileRegistryImage` and `pluginRegistryImage` unless it would differ from the value set by the operator of `airgapContainerRegistryHostname/airgapContainerRegistryOrganization/$IMAGE:$TAG`.  I don't think explicitly setting them would harm anything, other than if the image URL is incorrect

Reply:
Ah I didn't know that, thanks!

I've removed the two props here and added a sentence or two about what we expect for the registry images.